service/resources/phase_calculator.py

```
@@ -34,11 +34,8 @@ def on_post(self, _req, resp):
                     if _updated: # updates db
                         self.notifications(item, _phase) # sends notifications
                 commit_updates(self.session)
-        except Exception as err: # pylint: disable=broad-except
-            print("on_post error:")
-            print("{0}".format(err))
-            resp.status = falcon.HTTP_500
-            resp.body = json.dumps(jsend.error("{0}".format(err)))
+        except Exception: # pylint: disable=broad-except
```

Is this ignoring all exceptions? Can we be specific about the exception we'll get back from Twilio when the user has unsubscribed and just pass that one?

---

It needs to ignore exceptions to process the next record. Any exception will exit the app, any unprocessed record will get ignore. That's not the right behavior. 

I think we can implement a logger to log all responses from sending notifications.